### ISSUE TITLE
HTML "lang" & "dir" attributes are missing in the NVDA documentations for some languages

### ISSUE THREAD

**Issue by Mahmood-Taghavi** [nvaccess/nvda#8856]
NVDA documentations are HTML files that translated into many languages.
Unfortunately, "lang" & "dir" attributes are missing in NVDA docs for many languages.
The HTML documentations files are: "changes.html", "keyCommands.html", and "userGuide.html".
The "LANG" attribute helps NVDA to determine the language of documentation and always use the correct synthesizer.
The "DIR" attribute has the default value of "LTR" that is ok for most languages but for the right-to-left 
languages such as Persian (fa), Arabic (ar), and Hebrew (he), it must be set to "RTL" to show the documents in the correct view that be possible for reading. 
For example, currently the "nvda_2018.3.2\documentation\fa\keyCommands.html" have not even the DIR='RTL' attribute, while the corresponding file in the Arabic translation is ok.
Please fix this issue.
Thanks in advance

**Comment by ehollig** [comment]
CC @josephsl. Is it the responsibility of the translator to include the locale.t2tconf file for their particular language?

**Comment by ehollig** [comment]
Hello @smtsh. The "keyCommands.html" file is automatically generated when NVDA is compiled. I believe if the local

**Comment by smtsh** [comment]
Thanks to @ehollig comment, I found the below line is missing in the userGuide.t2t of Persian Language: 
%kc:includeconf: locale.t2tconf
I also checked the presence of locale.t2tconf in NVDA/user_docs for all languages and found that it is missing in most translations of the NVDA documents. Furthermore, it is missing in the original English documents so even the English documents have not HTML "lang" attribute.
So I think translators must be informed to fix for this issue!
The presence of the below line must check for "changes.t2t" in documents of all languages:
%!includeconf: locale.t2tconf
Morover, the presence of the below two lines must check for "userGuide.t2t" in documents of all languages:
%!includeconf: locale.t2tconf
%kc:includeconf: locale.t2tconf
Note: some translations (and even the English documents) have not a locale.t2tconf file to specify HTML "lang" attribute. But for example, Arabic translation with "ar" language code has a seemly perfect locale.t2tconf file.

**Comment by timveld** [comment]
Found this issue after noticing that Dutch documentation was read with an English voice in Edge... hope someone can get the language tags added.

**Comment by Adriani90** [comment]
@michaelDCurran I have had a look at this but I am not sure if there is a solution for this. In txt2tags I don'T find any configuration command for dir or lang atributes. I guess the html file is generated everytime the translation is updated, so we don't have any chance to apply the lang and dir atributes automatically during the conversion process. Right?
